intel-deflater.html

```diff
@@ -1,34 +0,0 @@
----
----
-<!doctype html>
-<html>
-	{% include head.html title="Intel Deflater" %}
-	<body>
-		<div class="wrapper">
-			{% include header.html %}
-
-			<h2>Intel Deflater</h2>
```

Comment:
@vdauwera I think @gspowley would probably be the best person to write an update for this.